app/src/main/java/com/duycuong/weather/utils/BindingUtils.java

```diff
@@ -0,0 +1,25 @@
+package com.duycuong.weather.utils;
+
+import android.databinding.BindingAdapter;
+import android.support.design.widget.TabLayout;
+import android.support.v4.view.PagerAdapter;
+import android.support.v4.view.ViewPager;
+import com.duycuong.weather.ui.screen.main.MainActivity;
```

Comment:
ERROR: Unused import: com.duycuong.weather.ui.screen.main.MainActivity. Use Ctrl+Shift+O (Eclipse) or Ctrl+Alt+O (Intellij) to sort imports